### ISSUE TITLE
Negative infoboxes are no longer styled

### ISSUE THREAD

**Issue by gnarea** [googlecodelabs/tools#627]
Consider this infobox in Markdown:

```
Negative
: It's important not to skip the command above if you're already using GCP. Otherwise, you'll end up modifying one of your existing projects.
```

It used to be rendered in an orange box and the word "Negative" wasn't displayed.

However, a regression must've been introduced because this is how it's rendered now:

![neg-infobox](https://user-images.githubusercontent.com/369863/130149895-7853b500-407e-47a0-a648-35b636625cf5.png)

This used to work fine until at least April.

**Comment by pmeijer-hva** [comment]
Workaround in markdown:

<aside class="special">
Your texts here and here
</aside>

**Comment by AHoxha19** [comment]
Same problem here. All negative and positive are not working :(

**Comment by TamimiGitHub** [comment]
Not sure if this is an issue or not, my positives and negatives are still broken. Do I need to go to an older version of Claat?

**Comment by KentyK** [comment]
I have the same problem here. This is a major bug.

**Comment by beaumoaj** [comment]
I am having the same issue.  The workaround enclosing Positive or Negative sections in `<aside class="positive">` or `<aside class="negative">` elements works but all the internal markup also needs to be written in HTML.
Is anyone working on fixing this is claat?  The issue does not seem to be assigned to anyone

**Comment by TamimiGitHub** [comment]
Bumping this

**Comment by beaumoaj** [comment]
Is anyone fixing this?  It's pretty annoying to have to switch to HTML inside a Positive or Negative aside element.  Why did the markdown options stop being supported in claat?  They are still in the format guide so they should be supported.

**Comment by TamimiGitHub** [comment]
I did some further investigation in this and went back to older versions of claat and confirmed that the issue is still there. I suspect that this broke when the css files hosting changed to `storage.googleapis.com/claat-public/*.css`. @mco-gh I cant find the old version of the css files that were hosted on `storage.googleapis.com/codelab-elements/*.css` to do a diff and investigate what changed. Can we look into the changes in these files to see what might be the issue?

**Comment by TamimiGitHub** [comment]
Looks like the change that fixes is the following
```
> aside negative
> Negative note here
```
```
> aside positive
> Positive note here
```

**Comment by tutek1** [comment]
Yes, this works in markdown. The documentation should be updated.
